ansible/sepolia.yml

```
@@ -52,3 +52,11 @@
       loop_control:
         loop_var: node
         index_var: idx
+
+    - include_role:
+        name: infra-role-geth-exporter
+        apply:
+          tags: always
+      tags: [ infra-role-geth-exporter ]
```

The tag is for operator convenience, it doesn't need to be that long.
```suggestion
      tags: [ geth-exporter ]
```

---

I agree, fixed.